pkg/agent/apis/types.go

```diff
@@ -72,6 +74,30 @@ func (r AntreaAgentInfoResponse) SortRows() bool {
 	return true
 }
 
+type FqdnCacheResponse struct {
+	fqdnName       string
+	ipAddress      net.IP
+	expirationTime time.Time
+}
+
+func (r FqdnCacheResponse) GetTableHeader() []string {
+	return []string{"FQDN", "ADDRESS", "EXPIRATION TIME"}
+}
+
+func (r FqdnCacheResponse) GetTableRow(maxColumn int) []string {
+	return []string{
+		r.fqdnName,
+		r.ipAddress.String(),
+		r.expirationTime.String(),
+	}
+}
+
+func (r FqdnCacheResponse) SortRows() bool {
+	return false
+}
+
+// maybe some helper funcs needed here to help parse through ^^^^
```

Comment:
Can you explain what this means

Reply:
was a dev note, removing now, as functionality has been added